configuration/reports/reportdescriptors/dataexports/sql/labOrdersReport.sql

```diff
@@ -0,0 +1,181 @@
+#set @startDate='1900-01-01';
+#set @endDate='2019-11-20';
+#a541af1e-105c-40bf-b345-ba1fd6a59b85 ZL
+#1a2acce0-7426-11e5-a837-0800200c9a66 Wellbody
+#0bc545e0-f401-11e4-b939-0800200c9a66 Liberia
+ 
+SELECT patient_identifier_type_id into @zlId from patient_identifier_type where uuid in ('a541af1e-105c-40bf-b345-ba1fd6a59b85' ,'1a2acce0-7426-11e5-a837-0800200c9a66','0bc545e0-f401-11e4-b939-0800200c9a66');
+SELECT person_attribute_type_id into @unknownPt FROM person_attribute_type where uuid='8b56eac7-5c76-4b9c-8c6f-1deab8d3fc47';
+SELECT encounter_type_id into @labResultEnc from encounter_type where uuid= '4d77916a-0620-11e5-a6c0-1697f925ec7b';
+SELECT order_type_id into @test_order from order_type where uuid = '52a447d3-a64a-11e3-9aeb-50e549534c5e';
+SELECT encounter_type_id into @specimen_collection from encounter_type where uuid = '39C09928-0CAB-4DBA-8E48-39C631FA4286';
+SELECT concept_id into @test_order from concept where uuid = '393dec41-2fb5-428f-acfa-36ea85da6666';
```

Comment:
Some of this could potentially be switched out to use variables from initialize_global_metadata()